docs/topics/unsigned-integer-types.md

```diff
@@ -9,7 +9,9 @@ In addition to [integer types](numbers.md#integer-types), Kotlin provides the fo
 
 Unsigned types support most of the operations of their signed counterparts.
 
->Changing type from unsigned type to signed counterpart (and vice versa) is a *binary incompatible* change.
+> Unsigned numbers are implemented as [inline classes](inline-classes.md) with the single storage property of the corresponding 
+> signed counterpart type of the same width. Thus, changing type from unsigned type to signed counterpart (and vice versa) 
```

Comment:
"Thus" doesn't seem logical here. I was proposing to connect these facts as a contrast.

Reply:
Now got it that this is the reason why someone could consider them compatible. Sorry, misunderstood. 
Does "Nevertheless" work here? [Commit.](https://github.com/JetBrains/kotlin-web-site/pull/2409/commits/297c3ff81fa91e1c1958044583a6516e0e7b13ba)